lib/OpenQA/WebAPI.pm

```diff
@@ -411,6 +411,9 @@ sub startup ($self) {
     $api_ra->delete('/isos/#name')->name('apiv1_destroy_iso')->to('iso#destroy');
     $api_ro->post('/isos/#name/cancel')->name('apiv1_cancel_iso')->to('iso#cancel');
 
+    # api/v1/webhooks
+    $api_ro->post('/webhooks/product')->name('apiv1_evaluate_webhook_product')->to('webhook#product');
```

Comment:
The route is called **product** but the PR description says `The route /api/v1/webhooks/iso is called by GitHub`.